ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -1,24 +1,32 @@
 <div class="replication-dashboard box is-sideless is-fullwidth is-marginless">
-  <h2 class="has-text-weight-semibold is-size-4 has-margin-bottom">Primary Cluster</h2>
+  <h2 class="has-text-weight-semibold is-size-4 has-bottom-margin-xs">Primary Cluster</h2>
   <p class="has-text-grey is-size-8">If you set a primary_cluster_addr when enabling replication, it will appear here.</p>
+
   {{#if model.dr.primaryClusterAddr}}
     <code>{{model.dr.primaryClusterAddr}}</code>
   {{else}}
     <code>no known primary cluster address</code>
   {{/if}}
 
-  <div class="selectable-card-container has-large-margin-top">
+  <div class="selectable-card-container has-top-margin-xl">
     {{yield (hash 
-      card=(
-        component componentToRender
-      )
+      card=(component componentToRender data=data)
     )}}
   </div>
-
+  
   {{yield (hash 
-    rows=(
-      component 'replication-table-rows' data=data
-    )
+    isSyncing=isSyncing
+    rows=(component 'replication-table-rows' data=data)
   )}}
 
+  <div class="documentation-link">
+    <p class="has-text-grey">We have additional timeseries telemetry that can be found 
+      <LearnLink 
+        @path="/vault/operations/monitor-replication#key-monitoring-metrics"
+        @class=""
```

Comment:
Is this to overwrite a default class?

Reply:
Oh, I think that's a leftover from testing.  Thank you for catching that.